### DISCUSSION TITLE
youtube is not playing

### DISCUSSION THREAD

**Discussion by Eddiemd78** [lavalink-devs/Lavalink#959] [Q&A]
Hey guys 
so the problem is when i put the bot on ytsearch or ytmsearch everything works fine bot will run , even it connects to vc and search correctly but the bot remains mute but when its on scsearch everything works correctly

**Comment by topi314** [comment]
I have a really hard time to understand what the actual issue is here
are you saying `ytsearch` or `ytmsearch` doesn't work?
if so can you include the lavalink logs please?
